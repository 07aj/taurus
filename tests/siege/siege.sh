#/bin/sh -xe
echo -e "[0;34mtext_mark,   2,HTTP/1.1,200,  0.39,   3372,/,0,2015-12-20 21:30:14[0m"
