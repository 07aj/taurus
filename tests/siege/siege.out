[0;34mtext_mark,   4,HTTP/1.1,200,  0.36,   3372,/,0,2015-12-20 21:30:14[0m
[0;34mtext_mark,   0,HTTP/1.1,200,  0.38,   3372,/,0,2015-12-20 21:30:14[0m
[0;34mtext_mark,   1,HTTP/1.1,200,  0.38,   3372,/,0,2015-12-20 21:30:14[0m
[0;34mtext_mark,   3,HTTP/1.1,200,  0.39,   3372,/,0,2015-12-20 21:30:14[0m
[0;34mtext_mark,   2,HTTP/1.1,200,  0.40,   3372,/,0,2015-12-20 21:30:14[0m
[0;34mtext_mark,   0,HTTP/1.1,200,  0.38,   3372,/,0,2015-12-20 21:30:14[0m
[0;34mtext_mark,   4,HTTP/1.1,200,  0.41,   3372,/,0,2015-12-20 21:30:14[0m
[0;34mtext_mark,   1,HTTP/1.1,200,  0.40,   3372,/,0,2015-12-20 21:30:14[0m
[0;34mtext_mark,   3,HTTP/1.1,200,  0.39,   3372,/,0,2015-12-20 21:30:14[0m
[0;34mtext_mark,   2,HTTP/1.1,200,  0.39,   3372,/,0,2015-12-20 21:30:14[0m
